### PR TITLE
[MAIL] Send mail with smtp without verifying SSL certificate

### PR DESCRIPTION
**Pull request by LouisOb**
config/mail.php: Added option to skip SSL certificate verification for self signed certificates

# Description
Using the current build the option to send mail with a self hosted SMTP server using a self signed SSL certificate is not possible. Adding the option to disable the verification process with the environmental variable `MAIL_VERIFY_SSL` allows to send mail.

Allowed values for  `MAIL_VERIFY_SSL` are `true` or `false`. Default is `true`

### Added
- config/mail.php
 ` 'verify_peer' => env('MAIL_VERIFY_SSL', 'true'),`

### CODE REVIEW

File: config/mail.php [hunk 1]
```diff
@@ -44,6 +44,7 @@
             'password' => env('MAIL_PASSWORD'),
             'timeout' => null,
             'local_domain' => env('MAIL_EHLO_DOMAIN'),
+            'verify_peer' => env('MAIL_VERIFY_SSL', 'true'),
```
Comment (alexjustesen) [inline config/mail.php:47]: Instead of passing `'true'` as a string can you update it to a boolean and pass `true` as the default?

**Review by alexjustesen — CHANGES_REQUESTED** [review]

File: config/mail.php (hunk 1 above)
Comment (alexjustesen) [reply config/mail.php:47]: Small change, once updated i'll merge. Thanks!

**Comment by alexjustesen** [comment]
Doc reference: https://laravel.com/docs/9.x/upgrade#symfony-mailer

File: config/mail.php (hunk 1 above)
Comment (LouisOb) [reply config/mail.php:47]: Will do.

Comment (LouisOb) [reply config/mail.php]: The new default value is commited.

**Review by alexjustesen — APPROVED** [review]

**Pull request merged**